main.go

```diff
@@ -103,8 +107,13 @@ func main() {
 		}
 	}
 
-	labels := Labels{}
+	// compile whiteListArg regex
+	labelWhiteList, err := regexp.Compile(whiteListArg)
+	if err != nil {
+		log.Fatalf("Error parsing white list regex: %s", err)
```

Comment:
Could you also log the argument value here? It could help people debug problems with shell escaping.